combat-trainer.lic

```diff
@@ -1200,6 +1200,8 @@ class SpellProcess
       @training_cyclic_timer = @training_cast_timer
     end
 
+    game_state.hide_on_cast = true if data['use_stealth']
```

Comment:
💡 Could this just as easily be `game_state.hide_on_cast = data['use_stealth']`?

Reply:
Yeah, I'll change that later tonight.